### PR TITLE
Add Scribe - Language Keyboards by @scribe-org

### PR DESCRIPTION
**Pull request by andrewtavis**
1. [x] Project URL:
2. [x] Update contents.json instead of README 
3. [x] One project per pull request
4. [x] Screenshot included 
5. [x] Avoid iOS or open-source in description as it is assumed
6. [x] Use this commit title format if applicable: Add app-name by @github-username
7. [x] Use approved format for your entry

### CODE REVIEW

**Comment by andrewtavis** [comment]
Let me know if I should use the exact order of the keys, or if anything else is needed. Thanks for this!

File: contents.json [hunk 1]
```diff
@@ -27350,6 +27350,28 @@
       ],
       "date_added": "Dec 4 2021",
       "suggested_by": "@dkhamsing"
+    },
+    {
+      "title": "Scribe - Language Keyboards",
+      "description": "Communicate with confidence using keyboards designed for language learners.”,
```
Comment (dkhamsing) [inline contents.json:27356]: Can you replace the `”` (smart quote?) at the end, should be `"`

File: contents.json [hunk 2]
```diff
@@ -27350,6 +27350,28 @@
...
+        "ipad",
+        "productivity",
+        "reference"
+      ],
+      "license": "gpl-3.0",
+      "source": "https://github.com/scribe-org/Scribe-iOS",
+      "date_added": "Dec 6 2021",
+      "suggested_by": "@andrewtavis",
+      "screenshots": [
+        "https://raw.githubusercontent.com/scribe-org/Scribe-iOS/main/Resources/GitHub/Images/scribeAppStoreImages.png",
```
Comment (dkhamsing) [inline contents.json:27372]: can you remove the comma at the end, thx!

File: contents.json (hunk 1 above)
Comment (andrewtavis) [reply contents.json]: Was wondering about the red readout, thanks!

**Comment by dkhamsing** [comment]
thx!

**Comment by andrewtavis** [comment]
Thank you as well! Have a nice rest of the week :)

**Pull request merged**